docs/operations/basic-cluster-tuning.md

```diff
@@ -40,11 +40,10 @@ The biggest contributions to heap usage on Historicals are:
 - Partial unmerged query results from segments
 - The stored maps for [lookups](../querying/lookups.md).
 
-A general rule-of-thumb for sizing the Historical heap is `(0.5GiB * number of CPU cores)`, with an upper limit of ~24GiB.
+A general rule-of-thumb for sizing the Historical heap is `(0.5GiB * number of CPU cores)`.
 
-This rule-of-thumb scales using the number of CPU cores as a convenient proxy for hardware size and level of concurrency (note: this formula is not a hard rule for sizing Historical heaps).
-
-Having a heap that is too large can result in excessively long GC collection pauses, the ~24GiB upper limit is imposed to avoid this.
+This is a starting point, not a hard rule for sizing Historical heaps.
+Note that with certain garbage collectors, having a large heap can result in excessively long GC pauses. For heaps larger than about 24GiB, we recommend using a collector that can handle large heaps, such as Shenandoah or ZGC.
```

Comment:
Are we confident that these GCs work well on large heap historicals in the wild? I am in total agreement about lifting the suggestion of a 24GiB cap on historical heaps, but wanted to confirm we are pushing the community somewhere they will find success. If indeed they work well out of the box, that is great, because I have experience having to tinker with G1 collector configs to get historical's running smoothly with large heaps.

Reply:
Not familiar with `Shenandoah` but at least `ZGC` works well for large heaps(like hundreds of GiB), especially the generational ZGC which is production ready in JDK 21.

Reply:
good to hear that about ZGC. I guess I'm not so concerned with pushing this out. Someone deploying open source druid and scaling historical heaps upwards of 100GiB sizes likely has the skills to navigate the best practices around choosing a collector and tuning it if needed.